Header header
string asifStatus
float64 TTS
float64 BTorthoBS
float64 hBackupEnd
float64 hSafetyNow
float64[4] xBackupEnd
float64 relax1
float64 relax2
float64 filterTimerUs